readme.md

```diff
@@ -3,70 +3,261 @@
-* So they found a creative way to tackle this that flips the Javascript MVC on its head and, in a way, shakes up everything you guys have learned about front-end web development in class so far.
 
-Here's a taste of what React is all about: [https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510)  
+> If you want to get a taste of what React's all about, [here's an introduction from React.js Conf 2015.](https://www.youtube.com/watch?v=KVZ-P-ZI6W4&feature=youtu.be&t=510)  
+
+### React in the MVC
+
+**React only concerns our "Views".**
+
+<details>
+  <summary>What is a "view" mean in front-end Javascript?</summary>
+
+  > Visual representations of our models - not the entire page.
+
+</details>
+
+React can coexist with Models and Controllers. The user can set those up however they see fit.
+* This means that React can also coexist with other Javascript frameworks. Let them handle the models and controllers, and have React sort out the views.
+
+# Initial Setup
+
+> **NOTE:** We're about to go through a lot of set-up that may not make 100% sense after going through it once. Don't worry if you don't understand the "Why?" quite yet. You will get more practice with this during the second React lesson.
+
+Our example app for this lesson is going to be a very simple blog. Let's start by creating a directory for the app and initializing `npm`...
+
+```bash
+$ mkdir wdi-weather && cd wdi-weather && npm init -y
+```
+
+> `npm init -y` accepts all the defaults that, without the flag, you would have to hit the return key to accept.
+
+Now, we're going to install **A LOT** of stuff. Bear with us for a second and run the following command in the terminal...
+
+```bash
+$ npm install --save react react-dom && npm install --save-dev html-webpack-plugin webpack webpack-dev-server babel-{core,loader} babel-preset-react
+```
+
+You'll know installation went A-OK if your `package.json` file looks like this...
+
+```json
+"dependencies": {
+  "react": "^15.0.1",
+  "react-dom": "^15.0.1"
+},
+"devDependencies": {
+  "babel-core": "^6.7.6",
+  "babel-loader": "^6.2.4",
+  "babel-preset-react": "^6.5.0",
+  "html-webpack-plugin": "^2.15.0",
+  "webpack": "^1.13.0",
+  "webpack-dev-server": "^1.14.1"
+}
+```
+
+Some of the terms in here should look familiar. We'll go over everything in more detail as we encounter them in the code throughout this lesson.
+* **React:** The library itself.
+* **React-DOM:** An additional module that allows us to update the DOM using React components.
+* **Webpack:** You learned about this "code bundler" in your [Build Tools lesson](https://github.com/ga-wdi-lessons/build-tools).
+* **Babel:** This one's actually new. We'll be using Babel to compile an HTML-like syntax called JSX into Javascript.
+
+> **Q:** What's the difference between `devDependencies` and `dependencies`?  
```

Comment:
:+1: Changed to details/summary.

Does the follow-up question make sense if there is no overlap between `dependencies` and `devDependencies`?